### ISSUE TITLE
Invalid widget or routine in CALL seems to crash client

### ISSUE THREAD

**Issue by rogerl50** [ArnoldSmith86/virtualtabletop#714]
A button with `clickRoutine`
```
  "clickRoutine": [
    {
      "func": "CALL",
      "widget": "foo"
    }
  ],
```
or
```
  "clickRoutine": [
    {
      "func": "CALL",
      "routine": "fooRoutine"
    }
  ],
```
where `foo` and `fooRoutine` are undefined seems to crash the client. I think it has something to do with mouse handling, because when I perform some mouse actions in the JSON editor, the screen seems to update. A console debug run makes it through the `CALL` code correctly.

**Comment by bjalder26** [comment]
I'm guessing part of this is related to issue #565.

**Comment by rogerl50** [comment]
Fixed (presumably by #893).
